platform/feature-api/communication/src/remote-pod/remote.ts

```diff
@@ -240,8 +240,8 @@ export class RemoteClientPod implements Pod {
                 return rpcClient.polyOut().writeFile(path, content, options)();
             }
 
-            importArchive(url: string): Promise<string> {
-                return rpcClient.polyOut().importArchive(url)();
+            importArchive(url: string, destUrl?: string): Promise<string> {
```

Comment:
One of the issues why I want to separate core from other changes, is to check that they are properly tested. This is a part of the code base that has a high coverage, and now we've simply created a different use case, with no tests to go with it. Since it's an optional argument, all tests will pass, of course, and the only real code there is is this one; still, all tests will be set with a null destUrl, which is not what we want.
Also, different use cases have been created. From your comments elsewhere, I think that the intention is to make the first call without a destUrl; the rest *with* them. So what happens if we do calls in a different sequence? Will that work?

Reply:
I will add tests. Calls in a different sequence are possible, but don't make sense, since the first call creates the `destUrl` for the rest of the calls.